dynawo/sources/Simulation/DYNSimulation.h

```diff
@@ -533,6 +533,9 @@ class Simulation {
   exportTimelineMode_t exportTimelineMode_;  ///< timeline's output mode
   std::string timelineOutputFile_;  ///< timeline's export file
 
+  std::string timestepsOutputFile_;  ///< timesteps's export file
+  int timestepsSteps_;  ///< timesteps's steps
```

Comment:
timesteps' steps and timesteps export file

Reply:
solved